### PR TITLE
Added more vdw radii

### LINKED ISSUE #2138: Topology table of vdw radii is pretty sparse
There is a dictionary of vdw radii in `MDAnalysis.topology.tables.vdwradii`.  It currently contains:

{'C': 1.5, 'F': 1.2, 'H': 0.4, 'N': 1.1, 'O': 1.05, 'S': 1.6, 'P': 1.6}

It should really cover more..

### PR DESCRIPTION
**Pull request by PicoCentauri**
Fixes #2138

Changes made in this Pull Request:
Added more vdw radii in a raw table and convert them using the `kv2dict` function.

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/topology/tables.py [hunk 1]
```diff
@@ -296,17 +298,76 @@ def kv2dict(s, convertor=str):
...
+#     J. Phys. Chem. 68 (3): 441-451. doi:10.1021/j100785a001.
+# [2] Rowland and Taylor (1996). "Intermolecular Nonbonded Contact Distances in Organic Crystal Structures:
+#                                 Comparison with Distances Expected from van der Waals Radii".
+#     J. Phys. Chem., 1996, 100 (18), 7384.7391. doi:10.1021/jp953141+.
+# [3] Mantina, et al. (2009). "Consistent van der Waals Radii for the Whole Main Group".
+#     J. Phys. Chem. A, 2009, 113 (19), 5806-5812. doi:10.1021/jp8111556.
+#------------ -----------
+# atomtype    r_vdw
+#------------ -----------
+H    1.10
```
Comment (richardjgowers) [inline package/MDAnalysis/topology/tables.py]: this value used to be 0.4

**Comment by PicoCentauri** [comment]
I also noticed these changes, but I first wanted to use the current literature values. In principle we could also introduce such a factor for [guessing the bonds](https://github.com/MDAnalysis/mdanalysis/blob/93c48f6061efc43ed2c9f3c042c931883c8465dd/package/MDAnalysis/topology/guessers.py#L139)...

**Comment by richardjgowers** [comment]
[We've already got a factor](https://github.com/MDAnalysis/mdanalysis/blob/93c48f6061efc43ed2c9f3c042c931883c8465dd/package/MDAnalysis/topology/guessers.py#L158) (which is from VMD I think) but with the new radii we'll need to find a new factor.  So with H being much smaller we'll have to go larger, but maybe this will cause trouble with other elements..

**Comment by tylerjereddy** [comment]
See also: http://redmine.gromacs.org/issues/2094#change-14100

I guess MDA isn't really used for solvation directly like GROMACS is, but obviously changing VDW radii can have a pretty massive effect for some calculations. When GROMACS changed their tables a few years ago I lost a few million particles in some virus simulation hydrations.

**Comment by orbeckst** [comment]
If the new radii + fudge factor gives better guesses than before then that's encouraging and provides motivation to change it. I am generally in favor of changing the radii. I don't know if these radii are outside of our bond guessers at all.

Perhaps you could add a more tests for the included systems where we have topology (bond) information. Basically, anything that has a 
- TPR
- PSF
- PRMTOP
- PDB with CONECT records

**Comment by richardjgowers** [comment]
@PicoCentauri thanks for the new tests, looks like you've got this working.  Where did 0.55 come from? Looks like the gromacs default is 0.57, then we can just say we're in line with that?  Also can you add a quick note to CHANGELOG to say that guess_bonds is different & hopefully better

**Comment by PicoCentauri** [comment]
> Where did 0.55 come from? Looks like the gromacs default is 0.57, then we can just say we're in line with that?

I started with the gromacs values, but I found the best agreement for a slightly smaller value. 0.56 also works for the two test cases, but for 0.57 the guesser will find too many bonds.

**Comment by richardjgowers** [comment]
@PicoCentauri this is ready to merge, it just needs a rebase if you can do that?

**Comment by PicoCentauri** [comment]
@richardjgowers done.

**Comment by richardjgowers** [comment]
@PicoCentauri awesome, thank you!

**Pull request merged**